src/versionbits.cpp

```diff
@@ -4,7 +4,20 @@
 
 #include "versionbits.h"
 
-ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex* pindexPrev, const Consensus::Params& params, ThresholdConditionCache& cache) const
+#include "consensus/params.h"
+
+const struct BIP9DeploymentInfo VersionBitsDeploymentInfo[Consensus::MAX_VERSION_BITS_DEPLOYMENTS] = {
+    {
+        /*.name =*/ "testdummy",
+        /*.gbt_force =*/ true,
+    },
+    {
+        /*.name =*/ "csv",
+        /*.gbt_force =*/ true,
```

Comment:
Is a binary forced/not-forced enough?

Taking segwit as an example, post-activation, what would ```gbt_force = true mean```? should I expect "segwit" to be always forced and transactions filtered? Forced if an commitment isn't required and an error otherwise? Isn't this a per-block property?

As gbt rules aren't necessarily static, I'm not sure that a simple flag makes sense. Or maybe I'm missing the bigger picture. Do you have that part coded up somewhere already, by chance?